<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::UnitConversionPropertyPage</class>
 <widget class="sf::PropertyPage" name="UnitConversionPropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnitConversionPropertyPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>&amp;Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout1">
        <item row="0" column="0">
         <widget class="QLabel" name="lblSystem">
          <property name="toolTip">
           <string>Select the unit system for the application.</string>
          </property>
          <property name="text">
           <string>S&amp;ystem</string>
          </property>
          <property name="buddy">
           <cstring>cbSystem</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbSystem">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblAsk">
          <property name="toolTip">
           <string>When enabled a dialog  pops up when missing an entry.</string>
          </property>
          <property name="text">
           <string>&amp;Ask</string>
          </property>
          <property name="buddy">
           <cstring>cbAsk</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cbAsk">
          <property name="text">
           <string>On/Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout2">
        <item row="0" column="0">
         <widget class="QLabel" name="lblEnableId">
          <property name="toolTip">
           <string>Enable or disable the unit conversion using a variable.</string>
          </property>
          <property name="text">
           <string>&amp;Enable ID</string>
          </property>
          <property name="buddy">
           <cstring>leEnableId</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leEnableId">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabConversions">
      <property name="statusTip">
       <string>Static unit conversions.</string>
      </property>
      <attribute name="title">
       <string>&amp;Conversions</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutTab1">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tbEditConversion">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbAddConversion">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbRemoveConversion">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="tvConversions">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFollowers">
      <property name="statusTip">
       <string>Dynamic unit conversion.</string>
      </property>
      <attribute name="title">
       <string>&amp;Followers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutTab2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tbEditFollower">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbAddFollower">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbRemoveFollower">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="tvFollowers">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sf::PropertyPage</class>
   <extends>QWidget</extends>
   <header>misc/qt/PropertyPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbSystem</tabstop>
  <tabstop>cbAsk</tabstop>
  <tabstop>leEnableId</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tbAddConversion</tabstop>
  <tabstop>tvConversions</tabstop>
  <tabstop>tbRemoveConversion</tabstop>
  <tabstop>tbAddFollower</tabstop>
  <tabstop>tvFollowers</tabstop>
  <tabstop>tbRemoveFollower</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
